src/amp-story-player/amp-story-player.js

```diff
@@ -14,9 +14,13 @@
  * limitations under the License.
  */
 
+import {AmpStoryPlayer} from './amp-story-player-impl';
 import {AmpStoryPlayerManager} from './amp-story-player-manager';
 
 self.onload = () => {
   const manager = new AmpStoryPlayerManager(self);
   manager.loadPlayers();
 };
+
+// eslint-disable-next-line no-undef
+globalThis.AmpStoryPlayer = AmpStoryPlayer;
```

Comment:
What is undefined?

Reply:
getting this linter error in VScode:
```
'globalThis' is not defined. eslintno-undef
```

I thought this was what you meant when you said I could safely suppress the eslint warning?

Reply:
This is unrelated. `globalThis` is a new identifier, just needs to be added to the predefined globals in `.eslintrc`. The one you'll need is to disable the `local/window-property-name`, which is fail once you rebase https://github.com/ampproject/amphtml/pull/28392 into this branch. (If you merge now, this will fail the linter).

Reply:
Oh got it, sent #28490 to add `globalThis` to `.eslintrc`

Reply:
Actually after merging #28490 I don't get any warnings, not even the `window-property-name` you mentioned 🧐

Reply:
Ahh, the lint rule doesn't operate on extension code for some reason: https://github.com/ampproject/amphtml/blob/7a69be3f4cfdcb3fd1d1895de609d3afb8f0d404/build-system/eslint-rules/window-property-name.js#L29-L33

Reply:
But this is actually under `src/`, not under `extensions/` 🤔